Makefile

```diff
@@ -227,11 +227,17 @@ build-binary-common:
 	cp COPYLEFT AllureOfTheStars
 	cp CREDITS AllureOfTheStars
 
-build-binary: build-binary-common
+build-binary-ubuntu: build-binary-common
 	cp AllureOfTheStarsInstall/bin/Allure AllureOfTheStars
 	dist/build/Allure/Allure --version > /dev/null; \
 	LH_VERSION=$$(cat ~/.Allure/stdout.txt); \
 	tar -czf Allure_$${LH_VERSION}_ubuntu-16.04-amd64.tar.gz AllureOfTheStars
 
+build-binary-macosx: build-binary-common
+	cp AllureOfTheStarsInstall/bin/Allure AllureOfTheStars
+	dist/build/Allure/Allure --version > /dev/null; \
+	LH_VERSION=$$(cat ~/.Allure/stdout.txt); \
+	tar -czf Allure_$${LH_VERSION}_macosx-amd64.tar.gz AllureOfTheStars
```

Comment:
Do you think it makes sense to mention the version of your OS? I guess it can be deduced with `ldd`. OTOH, it can save a person with too old OS X to have a chance of running that the effort of downloading and trying. What do you think?

Reply:
If we had many port maintainers with many OS X versions contributing their packages that would definitely make sense, but then, we don't have to worry ahead of time. :) Your call, really.

Reply:
Sure, I can add it so the name of the package is dependent on the OS version of the machine.